app/views/homepage/popular_links/_sidebar.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="sidebar-components_sidebar__border sidebar-component__button">
```

Comment:
I think this overcomplicating the CSS classes here - I would just give this element a class name of "sidebar-components". Then in the CSS we just need a rule to set the width of `.govuk-button` elements within that to 100%.